src/main/java/io/github/kobylynskyi/graphql/codegen/GraphqlCodegenMojo.java

```diff
@@ -18,165 +20,177 @@
-        this.generateEqualsAndHashCode = generateEqualsAndHashCode;
-    }
+  private void addCompileSourceRootIfConfigured() {
+    String path = outputDir.getPath();
+    getLog().info("Added the following path to the source root: " + path);
+    project.addCompileSourceRoot(path);
+  }
+
+  private static Map<String, String> convertToMap(Properties properties) {
+    Map<String, String> map = new HashMap<>();
+    properties.forEach((key, value) -> map.put((String) key, (String) value));
+    return map;
+  }
+
+  public String[] getGraphqlSchemaPaths() {
+    return graphqlSchemaPaths;
+  }
+
+  public void setGraphqlSchemaPaths(String[] graphqlSchemaPaths) {
+    this.graphqlSchemaPaths = graphqlSchemaPaths;
+  }
+
+  public File getOutputDir() {
+    return outputDir;
+  }
+
+  public void setOutputDir(File outputDir) {
+    this.outputDir = outputDir;
+  }
+
+  public Map<String, String> getCustomTypesMapping() {
+    return customTypesMapping;
+  }
+
+  public void setCustomTypesMapping(Map<String, String> customTypesMapping) {
+    this.customTypesMapping = customTypesMapping;
+  }
+
+  public String getPackageName() {
+    return packageName;
+  }
+
+  public void setPackageName(String packageName) {
+    this.packageName = packageName;
+  }
+
+  public String getApiPackageName() {
+    return apiPackageName;
+  }
+
+  public void setApiPackageName(String apiPackageName) {
+    this.apiPackageName = apiPackageName;
+  }
+
+  public String getModelPackageName() {
+    return modelPackageName;
+  }
+
+  public void setModelPackageName(String modelPackageName) {
+    this.modelPackageName = modelPackageName;
+  }
+
+  public String getModelNamePrefix() {
+    return modelNamePrefix;
+  }
+
+  public void setModelNamePrefix(String modelNamePrefix) {
+    this.modelNamePrefix = modelNamePrefix;
+  }
+
+  public String getModelNameSuffix() {
+    return modelNameSuffix;
+  }
+
+  public void setModelNameSuffix(String modelNameSuffix) {
+    this.modelNameSuffix = modelNameSuffix;
+  }
+
+  public Map<String, String> getCustomAnnotationsMapping() {
+    return customAnnotationsMapping;
+  }
+
+  public void setCustomAnnotationsMapping(Map<String, String> customAnnotationsMapping) {
+    this.customAnnotationsMapping = customAnnotationsMapping;
+  }
+
+  public String getModelValidationAnnotation() {
+    return modelValidationAnnotation;
+  }
+
+  public void setModelValidationAnnotation(String modelValidationAnnotation) {
+    this.modelValidationAnnotation = modelValidationAnnotation;
+  }
+
+  public boolean isGenerateApis() {
+    return generateApis;
+  }
+
+  public void setGenerateApis(boolean generateApis) {
+    this.generateApis = generateApis;
+  }
+
+  public boolean isGenerateEqualsAndHashCode() {
+    return generateEqualsAndHashCode;
+  }
+
+  public void setGenerateEqualsAndHashCode(boolean generateEqualsAndHashCode) {
+    this.generateEqualsAndHashCode = generateEqualsAndHashCode;
+  }
```

Comment:
Please fix the formatting with 4 spaces (as it was before).

Reply:
> Please fix the formatting with 4 spaces (as it was before).

Ok, sorry i had google style enable. I will change it. I prefer 4 spaces too.

Reply:
Done